File: HealthSysHub.Web.UI/Models/PharmacyOrderRequest.cs
﻿namespace HealthSysHub.Web.UI.Models
{
    public class PharmacyOrderRequest
    {
        public Guid? PharmacyOrderRequestId { get; set; }
        public Guid? PharmacyId { get; set; }
        public Guid? PatientPrescriptionId { get; set; }
        public Guid? HospitalId { get; set; }
        public Guid? PatientId { get; set; }
        public string? HospitalName { get; set; }
        public string? DoctorName { get; set; }
        public string? Name { get; set; }
        public string? Phone { get; set; }
        public string? Notes { get; set; }
        public string? Status { get; set; }
        public Guid? CreatedBy { get; set; }
        public DateTimeOffset? CreatedOn { get; set; }
        public Guid? ModifiedBy { get; set; }
        public DateTimeOffset? ModifiedOn { get; set; }
        public bool IsActive { get; set; }
    }
}

File: HealthSysHub.Web.Pharmacy.UI/src/app/models/pharmacy-order-request.ts
export interface PharmacyOrderRequest {
  pharmacyOrderRequestId: string;
  patientPrescriptionId?: string;
  hospitalId?: string;
  pharmacyId?: string;
  patientId?: string;
  hospitalName?: string;
  doctorName?: string;
  name?: string;
  phone?: string;
  notes?: string;
  status?: string;
  createdBy?: string;
  createdOn?: Date;
  modifiedBy?: string;
  modifiedOn?: Date;
  isActive: boolean;
}